typescript-forms/form-values.ts

```diff
@@ -0,0 +1,24 @@
+interface formElements extends HTMLFormControlsCollection {
+  name: HTMLInputElement;
+  email: HTMLInputElement;
+  message: HTMLTextAreaElement;
+}
+
+const $contactForm = document.querySelector<HTMLFormElement>('form');
```

Comment:
I'm not familiar with the syntax you're using above: `querySelector<...>`. Does it work?!
The reference answer does it this way: 
`const $contactForm = document.querySelector('#contact-form') as HTMLFormElement;`

Reply:
I guess both ways are correct. <HTMLFormElement> returns an element of the specific type.